lib/postprocessing/image.js

```diff
@@ -16,10 +16,163 @@ const gm = require("./gm-promisify");
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
+// list of supported output formats
+// maps our fmt ('png') to imagemagick format ('PNG') https://imagemagick.org/script/formats.php
+const SUPPORTED_OUTPUT_FMT = {
+    "png": "PNG",
+    "jpg": "JPEG",
+    "jpeg": "JPEG",
+    "tif": "TIFF",
+    "tiff": "TIFF",
+    "gif": "GIF"
+};
+
+// list of supported input formats (intermediate rendition)
+// in imagemagick format https://imagemagick.org/script/formats.php
+const SUPPORTED_INPUT_FMT = new Set([
+    "PNG",
+    "JPEG",
+    "JPEG",
+    "TIFF",
+    "TIFF",
+    "GIF",
+    "BMP"
+]);
+
+// minimal instructions which many workers can already handle themselves
+// const BASIC_INSTRUCTIONS = new Set([
```

Comment:
Remove it?

Reply:
Yeah, I wanted to have these fields explicitly listed somehow. These describe what "maximum" we can expect already be handled by workers. Right now they are covered by the 3 functions `isTargetFormat()`, `isExpectedSize()` and `hasJpegQuality()`. I wrote them down in this Set first, but then couldn't find a good way to "force" use of it, so I commented it out for now.

Maybe just turn this list of basic operations into a comment next to `EXTRA_INSTRUCTIONS`?

Reply:
Done in https://github.com/adobe/asset-compute-sdk/pull/49/commits/8e39ca05fb83026841e3cb361664a7f1515a76b5